docs/Project.toml

```diff
@@ -29,29 +30,31 @@ StructuralIdentifiability = "220ca800-aa68-49bb-acd8-6037fa93a544"
 Symbolics = "0c5d862f-8b57-4792-8d23-62f2024744c7"
 
 [compat]
-BifurcationKit = "0.3"
+BenchmarkTools = "1.5.0"
+BifurcationKit = "0.3.3"
 Catalyst = "13"
-DataFrames = "1"
-DiffEqParamEstim = "2.1"
-DifferentialEquations = "7.7"
-Distributions = "0.25"
-Documenter = "0.27"
-HomotopyContinuation = "2.6"
-Latexify = "0.15, 0.16"
-ModelingToolkit = "9.5"
-NonlinearSolve = "3.4.0"
-Optim = "1"
-Optimization = "3.19"
-OptimizationNLopt = "0.1.8"
-OptimizationOptimJL = "0.1.14"
-OptimizationOptimisers = "0.1.1"
-OrdinaryDiffEq = "6"
-Plots = "1.36"
-SciMLBase = "2.13"
-SciMLSensitivity = "7.19"
-Setfield = "1.1"
-SpecialFunctions = "2.1"
-SteadyStateDiffEq = "2.0.1"
-StochasticDiffEq = "6"
-StructuralIdentifiability = "0.5.1"
-Symbolics = "5.14"
+DataFrames = "1.6.1"
+DiffEqParamEstim = "2.2.0"
+DifferentialEquations = "7.13.0"
```

Comment:
I'd like to remove this one, but some old docs depend on it, so we can wait until those are fixed.